dfastmi/io/ObservableList.py

```diff
@@ -33,28 +33,54 @@
     ObservableList
 
 """
-class ObservableList:
+from typing import Iterator, List, Protocol, TypeVar, Generic
+
+
+T = TypeVar('T')
+
+# Define an observer interface
+class Observer(Generic[T]):
```

Comment:
should this be coherent to the other "interface" which start with I?

Reply:
Yes, updated.